content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -104,6 +104,31 @@ In this app, you are using the `react-native-global-props` library to set a cust
 
 Now, if you execute `react-native run-ios` or `react-native run-android` you should see the text with your custom fonts.
 
+![IOS](images/customize_fonts_result.png)
+
+## iOS "Unrecognized font family" error
+
+For some of you, maybe you are done (it used to work a few months ago), but if you are still getting the very common **Unrecognized font family** error, there are still some things you can check or do.
+
+First, you should check your **info.plist** file, located in the **ios/customize_fonts_react_native_tutorial** folder. This file must have a section with key **UIAppFonts** that contains an array of strings with your fonts. If this is missing, then you should add it:
```

Comment:
perhaps you can also mention that if you have more than one build target, you should remember to apply this change to their respective info.plist files